### PR TITLE
*For Testshops* - Adds/changes robots.txt to stop crawling

### PR DESCRIPTION
**Pull request by dpopov00**
For Testshop environments, robots.txt should be set to Disallow: /

### CODE REVIEW

File: src/Command/DevEnvCommand.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        $io = new SymfonyStyle($input, $output);
+        $robotsPath = $this->envPath."/robots.txt";
+
+        $fileExists = file_exists($robotsPath);
+        if($fileExists) {
+            $output->writeln('robots.txt exists in public folder');
+
+            $file = file_get_contents($robotsPath);
+            $content = "User-agent: *\nDisallow: /\n\n".$file;
```
Comment (shyim) [inline src/Command/DevEnvCommand.php:32]: Can you check here if this already is added to that file?

File: src/Command/DevEnvCommand.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+namespace Frosh\Tools\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class DevEnvCommand extends Command
+{
+    public static $defaultName = 'frosh:dev:env';
```
Comment (shyim) [inline src/Command/DevEnvCommand.php:12]:
```suggestion
    public static $defaultName = 'frosh:dev:robots-txt';
```

maybe?

File: src/Command/DevEnvCommand.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    public static $defaultDescription = 'For testshops - add/change robots.txt to stop crawling';
+    private string $envPath;
+
+    public function __construct(string $envPath)
+    {
+        parent::__construct();
+        $this->envPath = $envPath;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
```
Comment (shyim) [inline src/Command/DevEnvCommand.php:22]: Maybe a console option to remove this again?

File: src/Command/DevEnvCommand.php (hunk 1 above)
Comment (dpopov00) [reply src/Command/DevEnvCommand.php:32]: Of course, that's a good point :D

File: src/Command/DevEnvCommand.php (hunk 3 above)
Comment (dpopov00) [reply src/Command/DevEnvCommand.php:22]: I have to think a little bit about this, multiple entries can be inside, but it shouldn't be a problem

File: src/Command/DevEnvCommand.php (hunk 2 above)
Comment (dpopov00) [reply src/Command/DevEnvCommand.php:12]: Yeah, sure

**Comment by shyim** [comment]
Thanks. Whoopsiee. didnt saw that

**Pull request merged**